_assets/js/utils/constants.js

```diff
@@ -0,0 +1,11 @@
+// Pagination Constants:
+const paginationConstants = () => {
+    const constants = {
+        NUMBER_OF_RESULTS: 20,
+        SEARCH_ENDPOINT: "https://search.usa.gov/api/v2/search/i14y",
+        AFFILIATE: "justice-ada-beta",
+        ACCESS_KEY: "R9JA5YunOBaTGydUNm-oJjGCqKQ-zXsulsNskJVe5-c="  
+    } 
+    
+    return constants;
+}
```

Comment:
Is there an upside to making this a function?  Just curious.

Reply:
I did this as a function because initially I wanted to make this into an ES6 module with import/export statements. Since the current build pipeline doesn't allow that, I went with this design to approximate ES6 import/export statements.

Reply:
Works for me!